build/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
```

Comment:
Do we have to use this image as base? Or its more coming from operator-sdk?

Reply:
This is coming from the operator-sdk

Reply:
With the distroless effort also in progress, how will this be changed. @krishnadurai

Reply:
@nrchakradhar is there a link to "distroless effor" where I could understand what is the goal of that?

Reply:
https://github.com/kubeflow/kubeflow/issues/4589
This issue is about security. One of the items is distroless.
https://github.com/kubeflow/kubeflow/issues/4590

Reply:
Could we possibly switch to the distroless image for the final binary? We could use the [static-debian10](https://github.com/GoogleContainerTools/distroless#docker) image.
We've noticed that it reduces vulnerabilities for an image and it'd be great to make it a standard image for housing static binaries

Reply:
For some reason I can't pull the static-debian10 image, so I decided to go with base-debian10 image. I updated the dockerfile with multi-stage build to copy the built binary into the distroless  image now.

Reply:
Thanks!

Reply:
this can be resolved @Tomcli